Add GnuCOBOL documentation

jmerle opened the pull request:
Added a scraper for the GnuCOBOL documentation.

The documentation is licensed under the [GNU Free Documentation License](https://open-cobol.sourceforge.io/HTML/gnucobpg.html#Appendix-C-_002d-GNU-Free-Documentation-License).

GnuCOBOL doesn't have an official logo, so the default icon should be used.

j-f1 commented on lib/docs/filters/gnu_cobol/clean_html.rb:
```diff
@@ -0,0 +1,56 @@
+          end
+        end
+
+        # Remove all hr's after h4's
```
Can you also remove all hrs before h3s?

j-f1 replied:
Can you also remove all hrs before h3s?

jmerle commented:
Implemented everything you suggested. As for the x.y. and x.y.z. headers, they now look like this:
![](https://i.imgur.com/a3RgufA.png)

j-f1 commented on lib/docs/filters/gnu_cobol/clean_html.rb:
```diff
@@ -0,0 +1,72 @@
+          start_element.remove
+          start_element = next_element
+          next_element = start_element.nil? ? nil : start_element.next_element
+        end
```
This seems to remove the content of Appendix B, too. However, appendix B is just a list of all the reserved words, so it might be reasonable to remove it too.

jmerle commented:
I've added the logo you suggested. I considered adding it before but thought it wouldn't look good because of the details in it. Now that it's actually possible to generate the spritesheet programmatically I noticed that it wasn't so bad after all.

As for the `acronym` thing, all `acronym` tags are now converted to `span` tags so the underlining is removed.
